spec/parseBalanceQuery.spec.ts

```diff
@@ -15,6 +15,7 @@ describe("parseBalanceQuery", () => {
     const result = parseBalanceQuery({ _journal }, { name: "MyBook" });
     expect(result).to.deep.equal({
       book: "MyBook",
+      _journal: new Types.ObjectId(_journal),
       meta: { _journal },
```

Comment:
The `meta` property must be absent here. As well as in all the other unit tests this PR changes.

This object is literally the MongoDB query. The query will get 0 matches if it looks the same `_journal` ID in two fields simultaneously:
```
      _journal: new Types.ObjectId(_journal),
      meta: { _journal },
```

Reply:
we are only using meta in the snapshot, not in the query. please check balance changes in book file

Reply:
Still, see my main concern:
> Missing a unit test in balance.spec.ts where we would query a balance of a custom schema.

Reply:
Something like:
```js
      const snapshot = await getBestSnapshot({
        book: book.name,
        account: "Assets:Receivable",
        meta: { clientId: { $in: ["12345", "67890"] } },
       yourOwnCustomSchemaFields: "123412341234 whatever",
      });
```

Reply:
Fixed https://github.com/flash-oss/medici/pull/59/commits/ddb8c4e6806e3a11e0df4c9838b0cbb659e4cd38